Polyhedron/demo/Polyhedron/Scene_surface_mesh_item.h

```diff
@@ -29,15 +32,26 @@ class SCENE_SURFACE_MESH_ITEM_EXPORT Scene_surface_mesh_item
   typedef CGAL::Exact_predicates_inexact_constructions_kernel Kernel;
   typedef Kernel::Point_3 Point;
   typedef CGAL::Surface_mesh<Point> SMesh;
+  typedef SMesh FaceGraph;
   typedef boost::graph_traits<SMesh>::face_descriptor face_descriptor;
+  typedef boost::graph_traits<SMesh>::vertex_descriptor vertex_descriptor;
+  typedef boost::graph_traits<SMesh>::halfedge_descriptor halfedge_descriptor;
+  typedef SMesh::Property_map<vertex_descriptor,int> Vertex_selection_map;
+  typedef SMesh::Property_map<face_descriptor,int> Face_selection_map;
+  typedef SMesh::Property_map<halfedge_descriptor, bool> Halfedge_is_feature_map;
+  typedef SMesh::Property_map<face_descriptor, int> Face_patch_id_map;
+  typedef SMesh::Property_map<vertex_descriptor,int> Vertex_num_feature_edges_map;
```

Comment:
What I don't understand is how including 2 headers instead of one makes less dependency and why having a header including another header makes the compilation quicker. I agree that it seems cleaner to separate the code dealing with Surface_mesh from the code specific for the item, like we did in Polyhedron, but how does it make anything quicker, as everywhere Surface_mesh_type.h will be inclluded, so will Scene_surface_mesh_item ?